docs-2.0/20.appendix/releasenote.md

```diff
@@ -1,8 +1,58 @@
+- Support for error code display when a job fails. [#4067](https://github.com/vesoft-inc/nebula/pull/4067) 
+- The OS page cache can be disabled and the block cache and Nebula Graph storage cache can only be used in a shared environment, to avoid memory usage interference between applications. [#3890](https://github.com/vesoft-inc/nebula/pull/3890) 
+- Updated the default value of the KV separation threshold from 0 to 100. [#3879](https://github.com/vesoft-inc/nebula/pull/3879) 
+- Support for using gflag to set the upper limit of expression depth for a better fit of different machine environments. [#3722](https://github.com/vesoft-inc/nebula/pull/3722) 
+- Added a permission check for `KILL QUERY`. When the authorization is enabled, the GOD user can kill any query and the users with other roles can only kill queries that they own. [#3896](https://github.com/vesoft-inc/nebula/pull/3896) 
+- Support for more complier launchers, including distcc and sccache. [#3896](https://github.com/vesoft-inc/nebula/pull/3896) 
+- More dumping tables are supported with the meta dump tool. [#3870](https://github.com/vesoft-inc/nebula/pull/3870) 
+
+## Bugfix
+
+- Fixed the crash when using a function call as part of a filter in a `LOOKUP` statement. [#4111](https://github.com/vesoft-inc/nebula/pull/4111) 
+- Fixed the crash when there were non-indexed properties in an `IN` clause. [#3986](https://github.com/vesoft-inc/nebula/pull/3986) 
+- Fixed the storage service crash when concurrently scanning vertices and edges. [#4190](https://github.com/vesoft-inc/nebula/pull/4190) 
+- Fixed the crash when performing aggregation queries with patterns in a `MATCH` statement. [#4180](https://github.com/vesoft-inc/nebula/pull/4180) 
+- Fixed the crash when getting the JSON results of a `profile` query. [#3998](https://github.com/vesoft-inc/nebula/pull/3998) 
+- Fixed the crash when the `async` interface in the Lambda function finished running and the task in `threadManager` was not executed. [#4000](https://github.com/vesoft-inc/nebula/pull/4000) 
+- Fixed the `GROUP BY` output bug. [#4128](https://github.com/vesoft-inc/nebula/pull/4128) 
+- Fixed the bug that the version wasn't displayed with `SHOW HOSTS` sometimes. [#4116](https://github.com/vesoft-inc/nebula/pull/4116) 
+- Fixed the bug on parameters for `id(n) == $var`, `id(n) IN [$var]`, `id(n) == $var.foo.bar`, and `id(n) IN $var.foo.bar`. [#4024](https://github.com/vesoft-inc/nebula/pull/4024) 
+- Fixed the bug that an incorrect path direction occurred in `MATCH...WHERE`. [#4091](https://github.com/vesoft-inc/nebula/pull/4091) 
+- Fixed the bug that the result of referencing multiple `MATCH` variables in a `WHERE` clause was incorrect. [#4143](https://github.com/vesoft-inc/nebula/pull/4143) 
+- Fixed the optimizer bug. [#4146](https://github.com/vesoft-inc/nebula/pull/4146) 
+- Fixed the bug that the storage service failed to handle Raft snapshots. [#4019](https://github.com/vesoft-inc/nebula/pull/4019) 
```

Comment:
node

Reply:
maybe we can use service here instead of node.